docs/content/latest/quick-start/build-apps/go/ysql-yb-pgx.md

```diff
@@ -0,0 +1,576 @@
+    name: Go
+    identifier: go-1
+    weight: 552
+type: page
+isTocNested: true
+showAsideToc: true
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li>
+    <a href="../ysql-yb-pgx/" class="nav-link active">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - YugabyteDB PGX
+    </a>
+  </li>
+  <li>
+    <a href="../ysql-pgx/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - PGX
+    </a>
+  </li>
+  <li >
+    <a href="../ysql-pq/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - PQ
+    </a>
+  </li>
+  <li >
+    <a href="../ysql-pg/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - PG
+    </a>
+  </li>
+  <li >
+    <a href="../ysql-gorm/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - GORM
+    </a>
+  </li>
+  <li>
+    <a href="../ycql/" class="nav-link">
+      <i class="icon-cassandra" aria-hidden="true"></i>
+      YCQL
+    </a>
+  </li>
+</ul>
+
+This driver is a fork of [jackc/pgx](https://github.com/jackc/pgx) driver, with additional feature of connection load balancing.
+
+The following tutorial creates a simple Go application that connects to a YugabyteDB cluster using the [YugabyteDB pgx driver](https://pkg.go.dev/github.com/yugabyte/pgx), performs a few basic database operations — creating a table, inserting data, and running a SQL query — and then prints the results to the screen.
+
+The first example specifically demonstrates the connection load balancing using the `pgx.Connect()` API whereas the second example demonstrates the same using `pgxpool.Connect()` API.
+
+
+## Prerequisites
+
+This tutorial assumes that:
+
+- YugabyteDB is up and running. If you are new to YugabyteDB, you can download, install, and have YugabyteDB up and running within minutes by following the steps in [Quick start](../../../../quick-start/). Alternatively, you can use [Yugabyte Cloud](http://cloud.yugabyte.com/register) to get a fully managed database-as-a-service (DBaaS) for YugabyteDB.
+
+- [Go version 1.15](https://golang.org/dl/), or later, is installed.
+
+### SSL/TLS configuration
+
+You can find the details about the SSL/TLS configuration on the [YSQL PGX](../ysql-pgx/#ssl-tls-configuration) Go driver section. The same configuration is applicable to the YugabyteDB pgx driver as well.
+
+### Go YugabyteDB driver
+
+The [Go YugabyteDB driver package (`pgx`)](https://pkg.go.dev/github.com/yugabyte/pgx) is a fork of jackc/pgx Go driver. The current release of `pgx v4` requires Go modules.
+
+To install the package locally, run the following commands:
+
+```sh
+mkdir yb-pgx
+cd yb-pgx
+go mod init hello
+go get github.com/yugabyte/pgx/v4
+go get github.com/yugabyte/pgx/v4/pgxpool
+```
+
+## Write an application with pgx.Connect()
+
+Create a file `ybsql_hello_world.go` and copy the contents below.
+
+```go
+package main
+
+import (
+	"bufio"
```

Comment:
Can you use spaces instead of hard tabs? It's likely to render more reliably, unless Go _requires_ tabs instead of spaces.